includes/services.inc.php

```diff
@@ -129,7 +129,7 @@ function poll_service($service)
 
     // If we do not have a cmd from the check script, build one.
     if ($check_cmd == "") {
-        $check_cmd = $config['nagios_plugins'] . "/check_" . $service['service_type'] . " -H " . ($service['service_ip'] ? $service['service_ip'] : $service['hostname']);
```

Comment:
It is Great Idea to add
$config['force_ip_to_sysname'] = true;
Can we add same code for emails?
I still receive emails with IP hostnames. It is hard to recognize what is that. 
here is example of my email. 
![sysname_001](https://user-images.githubusercontent.com/17820730/27499853-eaffd3e2-5822-11e7-9298-7ea6e9578008.png)